packages/application-shell/src/components/quick-access/index.js

```diff
@@ -20,7 +21,21 @@ export default class QuickAccessTrigger extends React.Component {
     document.removeEventListener('keydown', this.handler);
   }
 
-  state = { isVisible: false };
+  // If for some reason an error is thrown, catch it and do not render QuickAccess
+  // to avoid crashing the entire app.
+  componentDidCatch(error, errorInfo) {
```

Comment:
This is just to prevent possible errors from quickaccess to crash the app

Reply:
👌